### ISSUE TITLE
Error 10 debugging

### ISSUE THREAD

**Issue by amit777** [EddyVerbruggen/cordova-plugin-googleplus#425]
I'm trying to get some insight if there is something obviously wrong with my environment or certificate/Oauth client ID config.  I have the plugin working for iOS fine. Android within my emulator is giving me hell.

Here is my environment:
cordova 7.0.1

Platforms:
cordova-android 6.2.3
cordova-ios 4.4.0

Plugins:
cordova-plugin-badge 0.8.1 "Badges"
cordova-plugin-console 1.0.7 "Console"
cordova-plugin-device 1.1.6 "Device"
cordova-plugin-fcm 2.1.2 "FCMPlugin"
cordova-plugin-googleplus 5.1.1 "Google SignIn"
cordova-plugin-inappbrowser 1.7.1 "InAppBrowser"
cordova-plugin-splashscreen 4.0.3 "Splashscreen"
cordova-plugin-statusbar 2.2.3 "StatusBar"
cordova-plugin-whitelist 1.3.2 "Whitelist"
ionic-plugin-keyboard 2.2.1 "Keyboard"

I'm getting Error 10. Here is a code snippet I've used to print out the webClientId and SigningFingerprint:

```
webClientId =  '235509812571-cl6h1bu4768vbc94lm6ihrjebr9uql9f.apps.googleusercontent.com';
window.plugins.googleplus.getSigningCertificateFingerprint(
    function (fingerprint) {
              console.log('##FINGERPRINT##');
              console.log(fingerprint);  //prints 98:91:E2:F2:7A:99:D2:94:45:1A:75:13:69:C3:A1:5C:F4:DB:AF:84
              console.log('##webClientId##');
              console.log(webClientId); //prints 235509812571-cl6h1bu4768vbc94lm6ihrjebr9uql9f.apps.googleusercontent.com
       } );

window.plugins.googleplus.login({
            scopes: 'email profile',
            webClientId: webClientId
          },          function(obj){
            console.log(obj);
          },          function(err){
            console.log(err); // I get the 10 code here
          });
```

The webClientId and Fingerprint match the info that I downloaded in google-services.json:
```
"oauth_client": [
        {
          "client_id": "235509812571-cl6h1bu4768vbc94lm6ihrjebr9uql9f.apps.googleusercontent.com",
          "client_type": 1,
          "android_info": {
            "package_name": "com.myapp.mobile",  // this is changed in this ticket only
            "certificate_hash": "9891e2f27a99d294451a751369c3a15cf4dbaf84"
          }
        },

```

**Comment by amit777** [comment]
Finally figured this out!  In case it's helpful to anyone else, here are high level notes on how i got it working:
1) The main thing I got wrong was that I was using teh Oauth2.0 Client_id associated with the certificate hash. This is of type "Android". This is the wrong client_id to use (even though the SHA1 hash is associated with it).  You must use a client_id of type "Web application"

2) I signed with my production keystore using this:
cordova emulate android --device -- --keystore=my-release-key.keystore --storePassword=mysecret --password=mysecret --alias=<alias_name>
